CRM/Utils/Money.php

```diff
@@ -201,7 +201,7 @@ protected static function formatLocaleNumeric($amount) {
    * @return string
    */
   protected static function formatLocaleNumericRounded($amount, $numberOfPlaces) {
-    return self::formatLocaleNumeric(round($amount, $numberOfPlaces));
+    return self::formatNumericByFormat($amount, '%!.' . $numberOfPlaces . 'i');
```

Comment:
@eileenmcnaughton just thinking it might be worthwhile somewhere having a comment about what this actually looks like i.e. '%!.2i' But this seems to work according to the docs (and here we go again on the money_format)